doc/plugins/lineprofiler.rst

```diff
@@ -0,0 +1,52 @@
+####################
+Spyder Line Profiler
+####################
+
+**Spyder-Line-Profiler** is a plugin to run the Python `line_profiler <https://pypi.python.org/pypi/line_profiler>`_.
```

Comment:
```suggestion
**Spyder-Line-Profiler** is a plugin to run the Python `line profiler <https://pypi.python.org/pypi/line_profiler>`_.
This package profiles the time that individual lines of code take to execute.
```

- There's no need to join `line` and `profiler` by an underscore.
- I also moved the line after the screenshot here because I think it makes more sense.